pipeline_steps/checkmarx.groovy

```diff
@@ -80,6 +80,10 @@ def scan(String scan_type, String repo_name, String exclude_folders){
                 // exception must propagate back to the retry call
                 throw e
             } //try
+
+            if (currentBuild.getResult() && currentBuild.getResult().isWorseThan(SUCCESS)) {
```

Comment:
The [documentation](https://rpc.jenkins.cit.rackspace.net/job/PM-Checkmarx_all-rpc-differ/pipeline-syntax/globals#currentBuild) doesn't list `currentBuild` as supporting this method but has others that are similar, has this been tested? This looks like something probably found in `currentBuild.rawBuild`.

Reply:
The documentation link is helpful. Pushed update to hopefully address the usage of this object.